tasks/ci.py

```diff
@@ -0,0 +1,63 @@
+import os
+import re
+
+from github import Auth, Github
+from invoke.tasks import task
+
+COMMIT_TITLE_REGEX = re.compile(r"\[test-infra-definitions\]\[automated\] Bump test-infra-definitions to ([a-z0-9]*)")
+
+
+@task
+def create_pr_and_close_stale_ones(ctx, branch: str, new_commit_sha: str, old_commit_sha: str):
+    if os.getenv("CI") != "true":
+        print("This task should only be run in CI")
+        return
+    if os.getenv("GITHUB_TOKEN") is None:
+        print("GITHUB_TOKEN is not set")
+        return
+
+    repo = Github(auth=Auth.Token(os.environ["GITHUB_TOKEN"])).get_repo("DataDog/datadog-agent")
+    pr_body = f"""
+This PR was automatically created by the test-infra-definitions bump task.
+
+This PR bumps the test-infra-definitions submodule to {new_commit_sha} from {old_commit_sha}.
+Here is the full changelog between the two commits: https://github.com/DataDog/test-infra-definitions/compare/{old_commit_sha}..{new_commit_sha}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
<3